src/lib/transform.js

```diff
@@ -62,6 +63,20 @@ function transformRelatedModel(field, prop) {
   return transformer;
 }
 
+function transformHTML(field) {
+  async function transformer(instance) {
+    const html = instance[field] || '';
+    const value = convert(html, { tables: true });
```

Comment:
looks like ```{ tables: true }``` is deprecated, they recommend 
```selectors: [ { selector: 'table', format: 'dataTable' } ]``` instead

https://github.com/html-to-text/node-html-to-text#options